emcc.py

```diff
@@ -2092,7 +2092,7 @@ def do_minify(): # minifies the code. this is also when we do certain optimizati
         # BINARYEN_METHOD with something that doesn't use the polyfill, then we don't need it.
         if not shared.Settings.BINARYEN_METHOD or 'interpret' in shared.Settings.BINARYEN_METHOD:
           logging.debug('integrating wasm.js polyfill interpreter')
-          wasm_js = open(os.path.join(binaryen_bin, 'wasm.js')).read()
+          wasm_js = open(os.path.join(shared.Settings.BINARYEN_ROOT, 'share', 'binaryen', 'wasm.js')).read()
```

Comment:
Hah, that's from the commit I marked as "[[TEMP - LOCAL ONLY]]" and then forgot about.
For reasons I didn't feel like looking into at the time, `wasm.js` got moved from where Emscripten expected it, so this test had started failing. My thought here was to locally fix the problem I was having locally, and then we'd fix this when updating the Emscripten that Binaryen uses for its tests?